spec/unit/facter/sensu_puppet_facts_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+require 'facter/sensu_puppet_facts'
+
+describe SensuPuppetFacts do
+  subject { SensuPuppetFacts }
+  before(:each) do
+    Puppet[:logdir] = '/tmp'
+    Puppet[:confdir] = '/tmp'
+    Puppet[:vardir] = '/tmp'
+    Puppet[:codedir] = '/tmp'
+    subject.add_facts
+  end
+  after(:each) do
+    Facter.clear
+    Facter.clear_messages
+  end
+
+  it 'should have puppet_hostcert defined' do
+    Puppet[:hostcert] = '/dne/host.pem'
```

Comment:
what's dne?

Reply:
Does Not Exist.  Just passing in something that doesn't exist for purposes of unit testing.